device/bgp/bgproutetarget.yaml

```diff
@@ -0,0 +1,23 @@
+components:
+  schemas:
+    Bgp.RouteTarget:
+      description: >-
+        BGP Route Target.
+      type: object
+      properties:
+        rt_type:
+          description: |-
+            Extended Community Type field of 2 Byte.
+             - as_2octet: Two-Octet AS Specific Extended Community (RFC 4360).
+             - ipv4_address: IPv4 Address Specific Extended Community (RFC 4360).
+             - as_4octet:  4-Octet AS Specific Extended Community (RFC 5668). 
+          type: string
+          enum:
+            - as_2octet
+            - ipv4_address
+            - as_4octet
+        rt_value:
```

Comment:
Please clarify:
- Please clarify how```rt_value``` will calculate when configured ```rt_type``` as ```ipv4_address```
- Normally we are using ```ASDOT```. But here we are using ``` Colon separated```. Can we consider Colon as DOT

Reply:
* `rd_value` examples for each `rt_type` has been updated in the description field.
* please dont consider colon as dot, because for `rt_type` `ipv4_address`, you need both dots and colon.